### DISCUSSION TITLE
HelloGitHub 徽章计划

### DISCUSSION THREAD

**Discussion by 521xueweihan** [wmjordan/PDFPatcher#212] [其它]
🎉 恭喜！您的[项目](https://hellogithub.com/repository/8deb5d9e48d54f9f83a1ad11d47b5bf6)已被 HelloGitHub 社区收录和推荐，现邀请您加入 HelloGitHub 徽章计划。加入后，您将获得以下特权：

- 社区认可: 徽章代表你的项目已通过 HelloGitHub 社区的严格筛选，并获得推荐。
- 提高曝光: 佩戴徽章后，你的项目将获得更多推荐流量，吸引更多用户和贡献者。
- 增强互动: 用户可通过徽章快速了解项目，并与之互动（点赞、评论、收藏）。
- 收集反馈: 获取广大用户的真实反馈，持续优化你的项目。
- 特殊标识: 认证后，你的评论将有显著标识，并获得置顶优先权。

📌 [立即点击](https://hellogithub.com/repository/8deb5d9e48d54f9f83a1ad11d47b5bf6/embed) 完成徽章佩戴，加入 HelloGitHub 徽章计划，让你的开源项目绽放更耀眼的光彩。

---

**HelloGitHub**，一个专注于发现、分享、推广开源项目的社区。自 2016 年创立以来，从每月一更的月刊发展到今天，已经成为一个拥有超过 **1 万** 用户的充满活力的开源社区。我们的足迹遍布多个内容平台，并获得了全网 **50 万** 粉丝的信赖与支持。

**Comment by 521xueweihan** [comment]
<img width="695" alt="image" src="https://github.com/wmjordan/PDFPatcher/assets/8255800/7a1a644e-9f8c-4bd5-bfa8-4b034f6d3a7b">

大爱无疆，向先生学习。

**Comment by wmjordan** [comment]
感谢您的关注和支持！

我看过您的网站，好像主要是以计算机语言来给项目做标签。
对于终端用户而言，他们不太感兴趣一个程序是用什么语言来开发的，也许再按用途来打个标签可以方便他们。
对于程序员来说，按用途来打标签也有助于他们去学习解决特定领域的问题。

这个项目已经有一些用户关注了。最后，为我另一个免费的项目打个广告：
**[给 C# 程序员开发的超能 Visual Studio 扩展插件 Codist](https://github.com/wmjordan/Codist)**

Codist 为程序员提供众多改善编辑环境的功能，例如极易配置的高级语法突出显示、信息量丰富的超级代码提示、顺手的智能工具条、C# 导航栏、优化的滚动条等等，全中文界面。可以说，C# 程序员一旦安装上它，就不会再卸载了。

**Reply by 521xueweihan** [reply]
![image](https://github.com/wmjordan/PDFPatcher/assets/8255800/45747609-ebe9-49d1-90f0-bee5cc4193ca)

用途的标签在项目介绍的下面，主要是功能标签和使用形式，比如 CLI、WEB、桌面应用等。

**Reply by wmjordan** [reply]
网站页面上好像没看到公众号的二维码呢？

**Reply by 521xueweihan** [reply]
@wmjordan 是的，网站的侧边栏没有放二维码。

关于 HelloGitHub 徽章的功能，我是考虑到 GitHub 并没有收集用户评价的功能，因为 issues 或者 discuss 是帖子的形式，解决即关，无法形成使用口碑的沉淀。

而且 HelloGitHub 的详情页无需作者花额外的心思维护，README 有入口顺手就可以点进来看看用户评价或回复下用户的使用评测之类的。

**Reply by 521xueweihan** [reply]
@wmjordan 先生，HelloGitHub.com 新增了极简的徽章样式哈，可自行选择样式佩戴。

**Reply by 521xueweihan** [reply]
<img width="693" alt="截屏2024-07-04 16 00 22" src="https://github.com/wmjordan/PDFPatcher/assets/8255800/a0ab6536-c1e7-43f3-a7be-f176f19c9e8e">
